.nvmrc

```diff
@@ -0,0 +1 @@
+17.9.0
```

Comment:
Should this perhaps be set to v16 so that this is on an LTS version of node? Or is there something about v17 that we want for these changes? 

Depending on when it's looking like these changes will be merged and released we might be able to even go up to the newest LTS version (v18) which will be released on the April 19th.

Reply:
Yeah, I did this because I thought it needed to be 17 for workspaces to work properly, but I think 16 should be ok. Good point re 18 though. We should probably migrate later.

I still need to work out the best way to run the node 12 tests though, because right now they're failing as they don't support workspaces at all.